bundle/Resources/config/parameters.yaml

```diff
@@ -68,3 +68,7 @@ parameters:
 
     ngsite.default.direct_download.content_types:
         - file
+
+    ngsite.default.tag_command_default_field_identifiers:
+        - tags
+        - main_topic
```

Comment:
Missing empty line at the end of the file.

Reply:
fixed in c45ef648